configure.ac

```diff
@@ -3825,6 +3825,19 @@ AC_COMPILE_IFELSE([AC_LANG_PROGRAM([[
    AC_MSG_RESULT(yes)
 ])
 
+AC_MSG_CHECKING([to see if the compiler supports __builtin_available])
```

Comment:
It is not necessary to test for this in a configure script, the following should work in a header file:

```
#ifdef __has_builtin
#if __has_builtin(__builtin_available)
#define HAVE_BUILTIN_AVAILABLE 1
#endif
#endif
```

Reply:
ok